buildscripts/kokoro/macos.cfg

```diff
@@ -15,7 +15,7 @@ env_vars {
 # We always build mvn artifacts.
 action {
   define_artifacts {
-    regex: "github/grpc-java/**/build/reports/**"
```

Comment:
why remove this line? We also want human readable (html) test reports right?

Reply:
They wouldn't generally be used, as the xml files will be processed directly by the viewers. e.g., see https://source.cloud.google.com/results/invocations/7572e440-611a-457e-b023-839457fddfef/targets

Reply:
You can see also files from the "Placer" link
https://fusion.corp.google.com/runanalysis/info/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/KOKORO/7572e440-611a-457e-b023-839457fddfef/0/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/Targets

Reply:
I think Sanjay's point is I'm excluding the HTML results; only the XML files are getting saved (to Placer, or elsewhere). For local usage, we view the HTML files instead of the XML files.

Reply:
The HTML files are also saved to Placer or GCS

e.g.

https://cnsviewer2.corp.google.com/placer/prod/home/kokoro-dedicated/build_artifacts/prod/grpc/java/master/presubmit/macos/5799/20220110-134420/github/grpc-java/xds/build/reports/tests/test/index.html

for the macos CI https://fusion.corp.google.com/runanalysis/info/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/KOKORO/d4761878-3b68-4879-b18a-493b7b73de60/0/prod%3Agrpc%2Fjava%2Fmaster%2Fpresubmit%2Fmacos/Targets

Reply:
Not with the change commented on here. Check the results of the CI here.

Reply:
@dapengzhang0, or are you saying you prefer looking through the results in placer than in the normal UIs (sponge, sponge2, source.cloud.google.com)? I agree that you could look at the files via cloud storage and placer, but I don't know why many people would do that, unless you really found the normal UIs lacking.

Reply:
My preference was reading the HTML is because
I'm not sure otherwise how to get human readable information easily from sponge link. If you want to see the STDERR, is the xml rendered in sponge log or you have to downloading the xml?

Reply:
I thought the general flow would be, click on the github "Details" link for macos, which takes you to [resultstore / source.cloud.google.com](https://source.cloud.google.com/results/invocations/5f4e0f80-b142-4baa-b3ae-e004572a3614/targets). Then you'd look at whatever test you're interested in (probably because it is failing. For example, [NettyTransportTest](https://source.cloud.google.com/results/invocations/5f4e0f80-b142-4baa-b3ae-e004572a3614/targets;query=nettytransporttest/github%2Fgrpc-java%2Fnetty%2Fbuild%2Ftest-results%2Ftest%2FTEST-io.grpc.netty.NettyTransportTest/tests). Then click Artifacts for the xml with stderr. That seems easier than digging through GCS and just already knowing it is there.

I don't see any way to make the HTML easier to find, as the HTML is per-package or per project (the index.html) and those don't align with the test results expected by the viewers.

Reply:
I see. That also looks good to me.